tools/obscuroscan_v2/frontend/src/views/HomeView.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+import { defineComponent } from 'vue'
+import SummaryItem from '@/components/SummaryItem.vue'
+import SummaryBatchesListItem from '@/components/SummaryBatchesListItem.vue'
+import SummaryRollupsListItem from '@/components/SummaryRollupsListItem.vue'
+
+export default defineComponent({
+  components: { SummaryRollupsListItem, SummaryBatchesListItem, SummaryItem }
+})
+</script>
+<template>
+  <el-container>
+    <el-main>
+      <summary-item />
+      <p>&nbsp;</p>
```

Comment:
These `<p>&nbsp;</p>` shouldn't be too hard to get rid of with css or whatever magic the element-plus styling has to offer.

Guessing these should just be like `margin-bottom: 5px` or something (and if it's breaking the FE library by adding scrollbar or something then it must have its own way of spacing things).